Views/home.stencil

```diff
@@ -13,16 +13,58 @@
     <p>Hack Cambridge</p>
   </section>
   <section class="LandingUpdateFeed">
-    <div class="LandingUpdateFeed__item">
-      <div class="PostCard">
-        <span class="PostCard__label">Workshop</span>
-        <h3 class="PostCard__title">Conquering Linux</h3>
+    <div><h1 class="LandingUpdateFeed__title Text--sectionHeading">Updates</h1></div>
+    <div class="LandingUpdateFeed__postContainer">
+      <div class="LandingUpdateFeed__item">
+        <div class="PostCard">
+          <div class="PostCard__photoBackground" style="background-image: url('/static/images/workshop.jpg')"></div>
```

Comment:
Is there a nice way to provide this URL on a card-by-card basis? I was hoping to find some kind of data-attribute-based way of doing it but couldn't find a way to do that.